text/0000-docs-team.md

```diff
@@ -0,0 +1,107 @@
+deciding who gets `r+` rights is important. A lack of clarity in this area has been unfortuante, and has led to a
+chicken and egg situation: without a documentation team, it's unclear how to be more involved in working on Rust's
+documentation, but without people to be on the team, there's no reason to form a team. For this reason, I think
+a small initial team will break this logjam, and provide room for new contributors to grow.
+
+# Detailed design
+[design]: #detailed-design
+
+The Rust documentation team will be responsible for all of the things listed above. Specifically, they will pertain
+to these areas of the Rust project:
+
+* The standard library documentation
+* The book and other long-form docs
+* Cargo's documentation
+* The Error Index
+
+Furthermore, the documentation team will be available to help with ecosystem documentation, in a few ways. Firstly,
+in an advisory capacity: helping people who want better documentation for their crates to understand how to accomplish
+that goal. Furthermore, monitoring the overall ecosystem documentation, and identifying places where we could contribute
+and make a large impact for all Rustaceans. If the Rust project itself has wonderful docs, but the ecosystem has terrible
+docs, then people will still be frustrated with Rust's documentation situation, especially given our anti-batteries-included
+attitude. To be clear, this does not mean _owning_ the ecosystem docs, but rather working to contribute in more ways
+than just the Rust project itself.
+
+We will coordinate in the `#rust-docs` IRC room, and have regular meetings, as the team sees fit. Regular meetings will be
+important to coordinate broader goals; and participation will be important for team members. We hold meetings weekly.
+
+## Membership
+
+* @steveklabnik, team lead
+* @GuillaumeGomez
+* @jonathandturner
+* @peschkaj
+
+It's important to have a path towards attaining team membership; there are some other people who have already been doing
+docs work that aren't on this list. These guidelines are not hard and fast, however, anyone wanting to eventually be a
+member of the team should pursue these goals:
+
+* Contributing documentation patches to Rust itself
+* Attending doc team meetings, which are open to all
+* generally being available on IRC to collaborate with others
+
+I am not quantifying this exactly because it's not about reaching some specific number; adding someone to the team should
+make sense if someone is doing all of these things.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+This is Yet Another Team. Do we have too many teams? I don't think so, but someone might.
+
+# Alternatives
+[alternatives]: #alternatives
+
+The main alternative is not having a team. This is the status quo, so the situation is well-understood.
+
+It's possible that docs come under the purvew of "tools", and so maybe the docs team would be an expansion
+of the tools team, rather than its own new team. Or some other subteam.
```

Comment:
I don't think so - I don't think the tools team have ever touched on a docs issue (other than the code of rustdoc).